modules/disk-uploader/pkg/vmexport/vmexport.go

```diff
@@ -24,15 +24,7 @@ const (
 	sourcePVC        string = "pvc"
 )
 
-var (
-	exportSources = map[string]struct{}{sourceVM: {}, sourceVMSnapshot: {}, sourcePVC: {}}
-)
-
-func CreateVirtualMachineExport(k8sClient *k8sv1.CoreV1Client, virtClient kubecli.KubevirtClient, exportSourceKind, exportSourceNamespace, exportSourceName string) error {
-	if !isValidExportSource(exportSourceKind) {
-		return fmt.Errorf("invalid export-source-kind: %s, must be one of vm, vmsnapshot, pvc", exportSourceKind)
-	}
-
+func CreateVirtualMachineExport(k8sClient kubernetes.Interface, virtClient kubecli.KubevirtClient, exportSourceKind, exportSourceNamespace, exportSourceName string) error {
```

Comment:
No need to pass ` kubernetes.Interface` as a parameter, because it is already part of `kubecli.KubevirtClient`